codecov/settings_dev.py

```diff
@@ -22,7 +22,7 @@
 
 CORS_ALLOW_CREDENTIALS = True
 
-GITHUB_CLIENT_ID = "3d44be0e772666136a13"
+GITHUB_CLIENT_ID = get_config("github", "client_id", default="Iv1.220bfc5d9b825038")
```

Comment:
Is the default a sensitive secret we'd want to hide?

Also I see some failing tests

Reply:
oh it didn't occur to me to run tests, wow. and email is so noisy i didn't see the notice. thanks for the catch!

the ID's not sensitive, there's a separate `client_secret` field which is + is in the yaml only